prime-router/src/main/kotlin/azure/BlobAccess.kt

```diff
@@ -21,10 +19,7 @@ import java.security.MessageDigest
 
 const val defaultBlobContainerName = "reports"
 
-class BlobAccess(
-    private val csvSerializer: CsvSerializer? = null,
-    private val hl7Serializer: Hl7Serializer? = null
-) : Logging {
+class BlobAccess : Logging {
```

Comment:
There is a possibility this can now be an object class, but it would suck to change all the references throughout the code.

Reply:
I wanna do that but I'm sure that would end up delaying the PR quite a bit...